Description: Bump UUID version to 1.10
Author: Soumya Ranjan Patnaik <soumyaranjan1812@gmail.com>
Last-Update: 2024-11-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trawldb/Cargo.toml
===================================================================
--- trawldb.orig/Cargo.toml
+++ trawldb/Cargo.toml
@@ -39,7 +39,7 @@ default-features = false
 version = "0.2.8"
 
 [dev-dependencies.uuid]
-version = "0.8.1"
+version = "1.10"
 features = [
     "serde",
     "v4",
